learning-guide/beginner.md

```diff
@@ -0,0 +1,30 @@
+# Beginner Guide
+
+Click the image to view on YouTube.
+
+| Title | Content | Length | 
+| --- | :-- | --- |
+| Full Roadmap to learn Blockchain development in 2021 | [![video](https://img.youtube.com/vi/ci_AIMCF-HA/0.jpg)](https://www.youtube.com/watch?v=ci_AIMCF-HA) | 12m 39s |
+| How to become a blockchain developer in 2021 | [![video](https://img.youtube.com/vi/OwSl2xwl2-w/0.jpg)](https://www.youtube.com/watch?v=OwSl2xwl2-w) | 1h 28m 20s |
+
+## Development Guide
+
+Click the image to view on YouTube.
+
+| Title | Content | Length | 
+| --- | :-- | --- |
+| The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
+
+## Blog content 
+
+About Blockchain, web 3 and somewhat related to Dapp.
+
+| Title | Content | 
+| :-- | --- |
+| The New Creator Economy - DAOs, Community Ownership, and Cryptoeconomics | <a href="https://dev.to/dabit3/the-new-creator-economy-daos-community-ownership-and-cryptoeconomics-lnl">View</a> |
+| The Complete Guide to Full Stack Ethereum Development | <a href="https://dev.to/dabit3/the-complete-guide-to-full-stack-ethereum-development-3j13">View</a>
+| What It Takes To Be A Blockchain Developer | <a href="https://www.techfunnel.com/information-technology/how-to-become-blockchain-developer/">View</a> |
+
+## Other
```

Comment:
| Title | Content | Length | 
| --- | :-- | --- |  

Should also be there

Reply:
In other part of the md file

Reply:
That's not a video, it is a blog so no need to put length.

Reply:
![12](https://user-images.githubusercontent.com/72812470/133135460-6ab84752-1c9d-407e-85de-c2bdc4e76b53.png)

Reply:
oh okay, the other category, I thought the blog ^^